### PR TITLE
[WIP] Updating sql server facts

### PR DESCRIPTION
**Pull request by zikalino**
##### SUMMARY
Adding missing SQL Server facts module.

##### ISSUE TYPE
 - New Module Pull Request

##### COMPONENT NAME
azure_rm_sqlserver_facts

##### ANSIBLE VERSION
2.6

### CODE REVIEW

File: lib/ansible/modules/cloud/azure/azure_rm_sqlserver_facts.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 DOCUMENTATION = '''
 ---
 module: azure_rm_sqlserver_facts
-version_added: "2.5"
+version_added: "2.7"
```
Comment (s-hertel) [inline lib/ansible/modules/cloud/azure/azure_rm_sqlserver_facts.py:19]: Was this meant to be nested under the new format option?

**Comment by Fred-sun** [comment]
@zikalino does this PR ready for review, please help confirm? Thanks!

**Comment by Fred-sun** [comment]
@zikalino Could you help to confirm the PR ready for review or not? Thanks!

File: lib/ansible/modules/cloud/azure/azure_rm_sqlserver_facts.py (hunk 1 above)
Comment (zikalino) [reply lib/ansible/modules/cloud/azure/azure_rm_sqlserver_facts.py:19]: yes

**Comment by Fred-sun** [comment]
@zikalino Could you help to confirm the PR ready for review or not? Thanks!

**Comment by Fred-sun** [comment]
@zikalino Could you help confirm that do you have finished this PR change? Thanks!

**Comment by Fred-sun** [comment]
kindly ping

**Comment by Fred-sun** [comment]
kindly ping

**Comment by Fred-sun** [comment]
kindly ping

**Comment by zikalino** [comment]
will create info module in 2.9

**Pull request closed without merging**